tools/xcodeproj-shims/xcodeproj-installer.sh

```diff
@@ -26,4 +27,7 @@ chmod -R +w "${tmp_dest}"
 # always trim three ../ from path, since that's "bazel-out/darwin-fastbuild/bin"
 sed -i.bak -E -e 's|([ "])../../../|\1|g' "${tmp_dest}/project.pbxproj"
 rm "${tmp_dest}/project.pbxproj.bak"
-rsync --recursive --quiet --copy-links "${tmp_dest}" "${dest}"
\ No newline at end of file
+rsync --recursive --quiet --copy-links "${tmp_dest}" "${dest}"
+
+mkdir -p "$dest/project.xcworkspace/xcshareddata/"
+mv "$(workspacesettings_xcsettings_short_path)" "$dest/project.xcworkspace/xcshareddata/"
```

Comment:
this should be a `cp`, rather than an `mv`, I think

Reply:
I'll add a comment!